spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -279,6 +279,11 @@ sendTimeout::
 +
 Default: `10000`
 
+useNativeEncoding::
```

Comment:
Well, this is Spring Clould Stream core property for all the producers.
See `spring.cloud.stream.bindings.[DESTINATION].use-native-encoding` - there is no `kinesis` prefix in the middle.

Having the property here may mislead users.

I would say the same, but as a general sentence in the end of this paragraph, after kinesis-specific producer properties. Something like:

> Also, if you'd like to produce a batch of records into Kinesis stream, the message payload must be as a `PutRecordsRequest` instance and general Spring Cloud Stream producer property `useNativeEncoding` must be set to `true`, so Spring Cloud Stream won't try to convert a `PutRecordsRequest` into a `byte[]`. The content of the `PutRecordsRequest` is now end-user responsibility.

Reply:
sure